tests/e2e/chain.go

```diff
@@ -0,0 +1,126 @@
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+
+	gaia "github.com/cosmos/gaia/v6/app"
+	"github.com/cosmos/gaia/v6/app/params"
+)
+
+const (
+	keyringPassphrase = "testpassphrase"
```

Comment:
shall we configure injeccting these via env vars? could help in automation of testing down the line for different senarios. what do you think?